src/index.html

```diff
@@ -9,6 +9,25 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="page__header">
+      <a href="./index.html" class="logo"><img src="./images/logo.png" alt="logo"></a>
```

Comment:
please wrap `<a>` tag with `<div>` - it is way easier to move around 😉

Reply:
You meen div > a > img?

Reply:
```suggestion
      <div class="some_class"><a href="./index.html" class="logo"><img src="./images/logo.png" alt="logo"></a></div>
```